fluent-bit-to-loki/pkg/config/config.go

```diff
@@ -219,5 +255,64 @@ func ParseConfig(cfg Getter) (*Config, error) {
 		res.ClientConfig.BackoffConfig.MaxBackoff = time.Duration(mab) * time.Second
 	}
 
+	// enable loki plugin buffering
+	buffer := cfg.Get("Buffer")
+	switch buffer {
+	case falseStr, "":
+		res.BufferConfig.Buffer = false
+	case trueStr:
+		res.BufferConfig.Buffer = true
+	default:
+		return nil, fmt.Errorf("invalid boolean Buffer: %v", buffer)
+	}
+
+	// buffering type
+	bufferType := cfg.Get("BufferType")
+	if bufferType != "" {
+		res.BufferConfig.BufferType = bufferType
+	}
+
+	// dque directory
+	queueDir := cfg.Get("DqueDir")
+	if queueDir != "" {
+		res.BufferConfig.DqueConfig.QueueDir = queueDir
+	}
+
+	// dque segment size (queueEntry unit)
+	queueSegmentSize := cfg.Get("DqueSegmentSize")
+	if queueSegmentSize != "" {
+		res.BufferConfig.DqueConfig.QueueSegmentSize, err = strconv.Atoi(queueSegmentSize)
+		if err != nil {
+			return nil, fmt.Errorf("impossible to convert string to integer DqueSegmentSize: %v", queueSegmentSize)
+		}
+	}
+
+	// dque control file change sync to disk as they happen aka dque.turbo mode
```

Comment:
I do not understand this comment. Could you rephrase it?

Reply:
I mean `dqueSync`. But this activate dque turbo where log entries are not immediately  stored on the file system